core/wiki/macros/tag-picker.tid

```diff
@@ -8,27 +8,34 @@ second-search-filter: [tags[]is[system]search:title<userInput>sort[]]
 \define delete-tag-state-tiddlers() <$action-deletetiddler $filter="[<newTagNameTiddler>] [<storeTitle>] [<tagSelectionState>]"/>
 
 \define add-tag-actions(actions,tagField:"tags")
-<$set name="tag" value={{{ [<__tiddler__>get[text]] }}}>
-<$list filter="[<saveTiddler>!contains:$tagField$<tag>!match[]]" variable="ignore" emptyMessage="""
-<$action-listops $tiddler=<<saveTiddler>> $field=<<__tagField__>> $subfilter="-[<tag>]"/>
-""">
-<$action-listops $tiddler=<<saveTiddler>> $field=<<__tagField__>> $subfilter="[<tag>]"/>
+<$set
+	name="tag"
+	value={{{ [<__tiddler__>get[text]] }}}
+><$list
+	filter="[<saveTiddler>!contains:$tagField$<tag>!match[]]"
+	variable="ignore"
+	emptyMessage='<$action-listops $tiddler=<<saveTiddler>> $field=<<__tagField__>> $subfilter="-[<tag>]"/>'
+><$action-listops
+	$tiddler=<<saveTiddler>>
+	$field=<<__tagField__>>
+	$subfilter="[<tag>]"/>
```

Comment:
I personally don't like constructions like this. It's the same as having the "semicolon in js at the beginning of the line"
I think it should look like this:

```
<$set name="tag" value={{{ [<__tiddler__>get[text]] }}}>
	<$list filter="[<saveTiddler>!contains:$tagField$<tag>!match[]]" variable="ignore" 
	emptyMessage='<$action-listops $tiddler=<<saveTiddler>> $field=<<__tagField__>> $subfilter="-[<tag>]"/>'>
		<$action-listops $tiddler=<<saveTiddler>> $field=<<__tagField__>> $subfilter="[<tag>]"/>

```
and so on. The proposal here is as unreadable for humans as if there wheren't any indentation at all

Reply:
All right. Fair enough. I've reverted the strange formatting. ...and now I'm being bold.

I've traced that macro back to the $keyboard widget it gets fed into. That wikitext is ONLY used for action widgets, and I cannot see how trimming the whitespace could ever cause a backward compatibility issue. I've gone ahead and added the whitespace trim to it.